wappsto/connection/communication.py

```diff
@@ -535,6 +549,34 @@ def send_thread(self):
 
             self.sending_queue.task_done()
 
+    def get_control(self, state):
+        """
+        Send get control state data.
+
+        Sends requests fot the data of control state.
+
+        Args:
+            state: State object referece.
+
+        """
+        self.wapp_log.info("Getting control value")
+        try:
+            local_data = self.rpc.get_rpc_state(
+                None,
+                state.parent_value.parent_network_id,
+                state.parent_value.parent_device_id,
+                state.parent_value.uuid,
+                state.uuid,
+                'control',
+                get=True
+            )
+            self.add_id_to_confirm_list(local_data)
+            self.create_bulk(local_data)
+        except OSError as e:
+            self.connected = False
+            msg = "Error sending control: {}".format(e)
```

Comment:
The error message should say the it failed to send a get request for the control value.